docs/standard/data/xml/mapping-the-object-hierarchy-to-xml-data.md

```diff
@@ -173,7 +173,17 @@ public class Sample
  **Output**  
   
 ```xml  
-<?xml version='1.0'?><!--This is a sample XML document --><!DOCTYPE Items [<!ENTITY number "123">]<Items><Item>Test with an entity: 123</Item><Item>test with a child element <more> stuff</Item><Item>test with a CDATA section <![CDATA[<456>]]> def</Item><Item>Test with a char entity: A</Item><--Fourteen chars in this element.--><Item>1234567890ABCD</Item></Items>  
+<?xml version='1.0'?>
```

Comment:
Since the code producing the output is using Console.Write instead of Console.WriteLine, the output doesn't contain newlines like you've added.

Reply:
ahm..sorry about that.